tests/artist-create.test.js

```diff
@@ -0,0 +1,37 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('create artist', () => {
```

Comment:
Indentation across this file is just 2 spaces, almost everywhere else it is 4. This should be brought in line with the rest of the project